qbraid/runtime/iqm/job.py

```
@@ -0,0 +1,217 @@
+
+    def _get_job(self, refresh: bool = False) -> Any:
+        if refresh or self._job is None:
+            self._job = self.session.get_job(self.id)
+        return self._job
+
+    @staticmethod
+    def _stringify_many(items: list[Any]) -> list[str]:
+        return [str(item) for item in items]
+
+    @classmethod
+    def _job_device_id(cls, job: Any) -> str | None:
```

Ruff (0.15.7)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `job`

(ANN401)